### PR TITLE
fix/price + feature_add_cart

### PR DESCRIPTION
**Pull request by Tkachenko01001**

### CODE REVIEW

**Comment by Tkachenko01001** [comment]
https://app.clickup.com/t/8695cyc68
@pqoqubbw

File: src/components/item-card/ItemCard.tsx [hunk 1]
```diff
@@ -58,14 +74,86 @@ const ItemCard: FC<Props> = ({ item }) => {
 
   const handleWeightClick = (
     e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
-    weight: string,
-    price: number
+    price: PriceResponse
   ) => {
+    console.log(price)
```
Comment (pqoqubbw) [inline src/components/item-card/ItemCard.tsx]: remove `console.log`

File: src/components/item-card/ItemCard.tsx [hunk 2]
```diff
@@ -58,14 +74,86 @@ const ItemCard: FC<Props> = ({ item }) => {
...
+
+        Notify.success('Товар добавлено в кошик!')
+      }
+    } catch (error) {
+      console.error('Помилка при додаванні товару до кошика:', error)
+      throw error
+    }
+  }
+
+  console.log(item)
```
Comment (pqoqubbw) [inline src/components/item-card/ItemCard.tsx]: same

**Review by pqoqubbw — CHANGES_REQUESTED** [review]
.

**Pull request merged**